frontend/views/pages/PayGroup.vue

```diff
@@ -20,65 +27,205 @@ page(
         i.icon-check(v-if='item.max === item.value')
         .has-text-1 {{ item.label }}
 
-  .c-container-empty(v-if='!hasPayments')
-    svg-contributions.c-svg
-
-    i18n.is-title-4.c-title(tag='h2') There are no pending payments yet!
-    i18n.has-text-1(tag='p') Once other group members add their income details, Group Income will re-distribute wealth amongst everyone.
-
-  section.card(v-else)
-    i18n.is-title-3(
-      tag='h3'
-      :args='{ month: paymentStatus.month }'
-    ) {month} contributions
-    i18n.has-text-1(
-      tag='p'
-      :args='{ amount: paymentTotal }'
-    ) {amount} in total
-
-    ul.c-payments
-      li.c-payments-item(
-        v-for='(payment, index) in paymentsDistribution'
+  i18n.p-descritpion.has-text-small.has-text-1(
+    tag='p'
+    data-test='openIncomeDetailsModal'
+    @click='openModal("IncomeDetails")'
+    :args='{ receive_or_send: needsIncome ? L("receiving") : L("sending"), r1: `<button class="link js-btnInvite">`, r2: "</button>"}'
+  ) You are currently {receive_or_send} mincome. You can change this at any time by updating your {r1}income details{r2}.
+
+  section.card
+    nav.tabs(
+      v-if='!needsIncome'
+      aria-label='navigation'
+      @click='open = false'
+    )
+      a.tabs-link(
+        v-for='(link, index) in tabItems'
         :key='index'
+        :class='{ "tabs-link-active": activeTab === link.url}'
+        :data-test='`link-${link}`'
+        @click='activeTab = link.url'
```

Comment:
This clickable element behaves like a "tab" that controls some other part of the UI, without changing pages. For that reason, it should be a `button`, not an `a`. And as a button, it should have `aria-expanded="true|false"` depending if it's active or not.

CSS-Tricks published very recently a [Complete Guide to Links and Buttons](https://css-tricks.com/a-complete-guide-to-links-and-buttons/). Check it out any time you have doubts between one or another.

Reply:
I'm still not sure if we should add parameter to the url for the tabs. But ok, for the time being I'll change to buttons

Reply:
That can be a possibility too, but not sure if it's worth at the moment as an MVP. Keep them as buttons is for now simpler